tools/integration_tests/readonly/create_objects.sh

```diff
@@ -0,0 +1,6 @@
+echo "This is from file a" >> a.txt
```

Comment:
can you move this scripts to readonly/testdata/

Reply:
fixed